### ISSUE TITLE
Problem seeing progress

### ISSUE THREAD

**Issue by Computerfan23** [Open-Travelers/OpenCrashWOC#7]
Hi there!
I just found out about this project. Really cool that you are doing this.
I am a blind gamer who uses screen reader software which allows me to read text and info on a webpage, type text using external keyboard etc. I have a problem viewing the progress of the project since screen readers have trouble handling certain formats. So it cannot tell me any numbers.
Do you have a way of measuring progress yet?

**Comment by td-9** [comment]
Hi! 
Thanks for the support, we don't have a real way of measuring progress at the moment, but I plan to add it soon
sorry for the late reply
